### PR TITLE
Jenkins integration

### PR DESCRIPTION
**Pull request by jcellary**
Added packages required to integrate sparrow with jenkins

### CODE REVIEW

File: apps/sparrow/settings/local.py [hunk 1]
```diff
@@ -35,5 +36,12 @@ def show_toolbar(request):
     'SHOW_TOOLBAR_CALLBACK': "sparrow.settings.local.show_toolbar",
 }
 
+JENKINS_TASKS = (
+    'django_jenkins.tasks.run_pep8',
+    'django_jenkins.tasks.run_pyflakes',
+    'django_jenkins.tasks.run_jshint',
+    'django_jenkins.tasks.run_csslint'
+)
```
Comment (harnash) [inline apps/sparrow/settings/local.py]: Shouldn't we include coverage as well?

**Comment by harnash** [comment]
It would be nice to include some word or two in README. How to run test from within Jenkins itself, requirements (Jenkins side) if any, etc.

**Comment by jcellary** [comment]
That's a great suggestion, but I haven't really finished setting up jenkins integration - I wanted to merge the dependencies first so that I can use them in any sparrow branch from jenkins. I will add the description in the readme in a separate PR, after I experiment more, does that sound fair?

File: apps/sparrow/settings/local.py (hunk 1 above)
Comment (jcellary) [inline apps/sparrow/settings/local.py]: Coverege is calculated when you run ./manage.py jenkins --enable-coverage

Comment (harnash) [inline apps/sparrow/settings/local.py]: Right. I somehow thought it was separate action, probably because I was thinking about SLOCCount.

**Comment by harnash** [comment]
It's fine with me. Until the initial release of course :)

**Comment by harnash** [comment]
As initial integration we can leave it as it is. I would like it to be split (settings and requirements) into different environment (testing).

**Pull request merged**